commands/validate.ts

```diff
@@ -0,0 +1,52 @@
+import Course from "../lib/models/course";
+import BaseCommand from "./base";
+import RuntimeTester from "../lib/testers/runtime-tester";
+
+export default class ValidateCommand extends BaseCommand {
```

Comment:
Can we name this `BuildAndRun` too? Weird to have a command name that's different from the command class